tr/about/index.md

```diff
@@ -166,12 +166,11 @@ bahsedilmiştir:
   yükleyebilirsiniz.
 
 * Ruby işletim sisteminden bağımsız iş parçacığı özelliği sunar.Yani
-  işletim sisteminin desteklemesine bakmaksızın, MS-DOS üzerinde olsanız
-  bile çoklu iş parçacıkları kullanabilirsiniz!
+  işletim sisteminin desteklemesine bakmaksızın!
 
 * Ruby yüksek taşınabilirliğe sahiptir. GNU/Linux üzerinde geliştirilmiş
   olsa dahi, UNIX’in bir çok çeşidi, macOS,
-  Windows, DOS, BeOS, OS/2, vb. üzerinde çalışmaktadır.
+  Windows, vb. üzerinde çalışmaktadır.
```

Comment:
@jonjensen `vb.` means is `etc` in Turkish. is there any operating systems except Unix distributions, macOSX and Windows? In English it is `It runs under Linux and other UNIX-like operating systems, macOS and Windows` there is no etc so we can use `vb.` here. It should be in Turkish `Ruby yüksek taşınabilirliğe sahiptir. GNU/Linux üzerinde geliştirilmiş olsa dahi, UNIX’in bir çok çeşidi, macOS ve Windows üzerinde çalışmaktadır.`

Reply:
Ok, makes sense.